src/astrobleme/logic/GameLoop.java

```diff
@@ -37,30 +37,39 @@
     private final GraphicsContext context;
     private float interval;
     private boolean running;
-    private boolean paused;
-    private boolean keyIsPressed;
+    private boolean stopped;
```

Comment:
I named it paused because it can be potentially extended to support a pause button. Stop takes away that possibility IMO.